app/assets/javascripts/modules/thumb_slider.js

```diff
@@ -0,0 +1,97 @@
+(function( global ) {
+  'use strict';
+  var Module = (function() {
+    var selector;
+
+    var containerTemplate =
+      jQuery(
+        '<div class="sul-embed-thumb-slider-container"></div>'
+      );
+
+    var toggleControl =
+      jQuery(
+        '<div class="sul-i-navigation-show-more-1 sul-embed-thumb-slider-open-close"' +
+          ' aria-expanded="true" aria-label="toggle thumbnail viewer">' +
+        '</div>'
+      );
+
+    var thumbList =
+      jQuery(
+        '<div class="sul-embed-thumb-slider" style="display:none">' +
+          '<ul></ul>' +
+        '</div>'
+      );
+
+    function hideInitialMediaObjects() {
```

Comment:
I guess I'm wondering if "Media" should be in this name, if you want this to be general:  hideInitialThumbs() ???